docs/howto/manage-configurations.md

```diff
@@ -30,21 +49,28 @@ In the `src/charm.py` file of the charm project, in the `__init__` function of t
 self.framework.observe(self.on.config_changed, self._on_config_changed)
 ```
 
-Then, in the body of the charm definition, define the event handler. Here you may want to read the current configuration value, validate it (Juju only checks that the *type* is valid), and log it, among other things. Sample code for an option called `server-port`, with type `int`, and default value `8000`:
+Also in the `__init__` function, load the config into the config class that you
+defined. Pass `errors='blocked'` to have the charm exit after setting a blocked
+status if the configuration doesn't validate against the class you defined.
```

Comment:
Maybe add something like "The default is `errors='raise'`, which means that `load_config` will raise the validation error."